### PR TITLE
feat(AUTH-CPC-502): auth_panel_getAll and delete user

### PR DESCRIPTION
**Pull request by Jaylapuck**
JIRA: link to jira ticket
## Context:
https://champlainsaintlambert.atlassian.net/jira/software/c/projects/CPC/boards/16/backlog?view=detail&selectedIssue=CPC-502&issueLimit=100
Added a UI panel to display the user.
## Changes
Mostly angularJS coding, API endpoints were already implemented.
## Before and After UI (Required for UI-impacting PRs)
image

### CODE REVIEW

**Review by JordanAlbayrak — APPROVED** [review]
I see that you readded the jacococ coverage to 90% and the rest lgtm, poe.

File: auth-service/src/main/java/com/petclinic/auth/User/UserServiceImpl.java [hunk 1]
```diff
@@ -83,6 +79,11 @@ public User getUserById(long id) {
         return userRepo.findAll(of);
     }
 
+    @Override
+    public List<User> findAllWithoutPage() {
+        return userRepo.findAll();
+    }
+
```
Comment (Fube) [inline auth-service/src/main/java/com/petclinic/auth/User/UserServiceImpl.java:86]: Why is this needed?

File: api-gateway/src/main/java/com/petclinic/bffapigateway/domainclientlayer/AuthServiceClient.java [hunk 2]
```diff
@@ -70,19 +70,17 @@ public AuthServiceClient(
...
+                .uri(authServiceUrl + "/users/{userId}", userId)
+                .body(just(model), Register.class)
                 .accept(MediaType.APPLICATION_JSON)
-                .retrieve().bodyToMono(UserDetails.class);
+                .retrieve()
+                .onStatus(HttpStatus::is4xxClientError,
+                        n -> rethrower.rethrow(n,
+                                x -> new GenericHttpException(x.get("message").toString(), BAD_REQUEST))
+                        )
+                .bodyToMono(UserDetails.class);
```
Comment (Fube) [inline api-gateway/src/main/java/com/petclinic/bffapigateway/domainclientlayer/AuthServiceClient.java:83]: I cannot seem to find any tests related to this.
Can you please link me to them?
